### PR TITLE
Lscvs

### PR DESCRIPTION
**Pull request by gcsantos-gpa**
Update for OpenXDA to reflect changes needed for LSCVS v2.0

### CODE REVIEW

**Review by clackner-gpa — APPROVED** [review]

File: Source/Applications/openXDA/openXDA/openXDA.csproj [hunk 1]
```diff
@@ -192,6 +194,10 @@
     <Reference Include="Microsoft.Owin.Security">
       <HintPath>..\..\..\Dependencies\GSF\Microsoft.Owin.Security.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.SqlServer.Types, Version=11.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>C:\Program Files\Microsoft SQL Server\150\DTS\Extensions\Common\Microsoft.SqlServer.Types.dll</HintPath>
+    </Reference>
```
Comment (StephenCWills) [inline Source/Applications/openXDA/openXDA/openXDA.csproj:200]: Maybe it has something to do with this. It seems like this should be a NuGet package, but somehow the HintPath is referencing something in Program Files, and the DLL was manually added to `Dependencies\SQL`. Can someone verify that these dependencies are needed and that they're being referenced correctly?

**Review by StephenCWills — CHANGES_REQUESTED** [review]
Just a few things I noticed while skimming through the changes.

File: Source/Applications/openXDA/openXDA/openXDA.csproj [hunk 2]
```diff
@@ -158,6 +158,8 @@
     <Reference Include="log4net, Version=2.0.12.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a, processorArchitecture=MSIL">
       <HintPath>..\..\..\Dependencies\NuGet\log4net.2.0.12\lib\net45\log4net.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.AspNet.Scaffolding.12.0, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
+    <Reference Include="Microsoft.AspNet.Scaffolding.EntityFramework.12.0, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
```
[the remark this answers is not in the document]
Comment (clackner-gpa) [reply Source/Applications/openXDA/openXDA/openXDA.csproj:162]: That's all related to the `Microsoft.SqlServer.Types.dll` dependencies @gcsantos-gpa is trying to get that fixed.

File: Source/Libraries/openXDA.Model/openXDA.Model.csproj [hunk 3]
```diff
@@ -46,6 +46,10 @@
     <Reference Include="Ionic.Zlib">
       <HintPath>..\..\Dependencies\DotNetZip\Ionic.Zlib.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.SqlServer.Types, Version=11.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\Dependencies\SQL\Microsoft.SqlServer.Types.dll</HintPath>
+    </Reference>
```
[the remark this answers is not in the document]
Comment (gcsantos-gpa) [reply Source/Libraries/openXDA.Model/openXDA.Model.csproj]: Fixed, all of the dependencies you pointed out were used at one point during the creation process but not now. They have all been removed.

File: Source/Libraries/FaultData/DataOperations/LSCVSDataOperation.cs [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+        }
+
+        /// <summary>
+        /// Gets AntiForgeryToken from openXDA
+        /// TODO: Refactor this
+        /// </summary>
+        /// <returns>string token</returns>
+        public string GenerateAntiForgeryToken() //Todo: When refactoring, change this to a static class...
+        {
+            using (HttpClient client = new HttpClient())
```
[the remark this answers is not in the document]
Comment (gcsantos-gpa) [reply Source/Libraries/FaultData/DataOperations/LSCVSDataOperation.cs]: This TODO refers to moving the function into GSF, which will be done later, but you are correct in that this needs to be changed to be static. It has been done, thank you for the assistance in resolving this issue.

**Pull request merged**